src/sql/connection.py

```diff
@@ -336,3 +336,15 @@ def _close(cls, descriptor):
 
     def close(self):
         self.__class__._close(self)
+
+    def get_curr_connection_info(self):
```

Comment:
let's make this a private method `_get_curr_connection_info`

question: is `self.current` guaranteed to be the engine that will execute the SQL statement? because users might switch the connection https://jupysql.ploomber.io/en/latest/howto.html?highlight=switch#switch-connections

Reply:
I'd better to add the test case to verify this

Reply:
1. Fix to _get_curr_connection_info in [ed2fda3](https://github.com/ploomber/jupysql/pull/126/commits/ed2fda32b9a757630a3ac44d658db65a3bb3edec)
2. self.current is guaranteed to be the engine? I think so, since in [connection.py](https://github.com/ploomber/jupysql/blob/master/src/sql/connection.py#L205) everytime there is the new command, it will reset the current connect as self. I also added test case to verify in [bbad823](https://github.com/ploomber/jupysql/pull/126/commits/bbad823f3794e41ae3331f1593799c2bfd0c990f)